modules/drivers/snowflake/src/metabase/driver/snowflake.clj

```diff
@@ -254,6 +259,22 @@
   [driver [_ value _unit]]
   (hx/->time (sql.qp/->honeysql driver value)))
 
+(defmethod sql.qp/->honeysql [:snowflake :convert-timezone]
+  [driver [_ arg target-timezone source-timezone]]
+  (let [clause       (sql.qp/->honeysql driver arg)
```

Comment:
I think `clause` as a name is misleading. `arg` might not be an MBQL clause as it could be a literal. Reusing `arg` would be an improvement.

Reply:
I'll call it hsql-form.